kustomize/commands/localize/localize.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2022 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package localize
+
+import (
+	"fmt"
+	"io"
+	"log"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kustomize/api/konfig"
+	lclzr "sigs.k8s.io/kustomize/api/krusty/localizer"
+	"sigs.k8s.io/kustomize/kyaml/errors"
+	"sigs.k8s.io/kustomize/kyaml/filesys"
+)
+
+const cmdName = "localize"
+const numArgs = 2
+const alphaWarning = `Warning: This is currently an alpha command.
+`
+
+type arguments struct {
+	target string
+	dest   string
+}
+
+type flags struct {
+	scope string
+}
+
+// NewCmdLocalize returns a new localize command.
+func NewCmdLocalize(fs filesys.FileSystem, writer io.Writer) *cobra.Command {
+	log.SetOutput(writer)
+	var f flags
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("%s [target [destination]]", cmdName),
+		Short: "Creates localized copy of target kustomization root at destination",
+		Long: fmt.Sprintf(`Creates copy of target kustomization directory
+or versioned URL at destination, where remote references in the original are 
+replaced by local references to the downloaded remote content.
+
+If target is not specified, the current working directory will be used. 
+Destination is a path to a new directory in an existing directory. If 
+destination is not specified, a new directory will be created in the current 
+working directory. 
+
+For details, see: https://kubectl.docs.kubernetes.io/references/kustomize/cmd/
+
+Disclaimer:
+This command does not yet localize helm or KRM plugin fields. This command also
+alphabetizes kustomization fields in the localized copy.
+
+%s`, alphaWarning),
```

Comment:
We should probably mark this as alpha the same way we do the other alpha commands. For example:

```
$ kustomize cfg -h
Commands for reading and writing configuration

Usage:
  kustomize cfg [command]

Available Commands:
  cat           [Alpha] Print Resource Config from a local directory.
  count         [Alpha] Count Resources Config from a local directory.
  grep          [Alpha] Search for matching Resources in a directory or from stdin
  tree          [Alpha] Display Resource structure from a directory or stdin.
```

Reply:
Done. I added `[Alpha]` to both the short and long descriptions of the command.